src/FaceRecognizer.cc

```diff
@@ -1,15 +1,19 @@
-#include "FaceRecognizer.h"
 #include "OpenCV.h"
 
-#if CV_MAJOR_VERSION >= 3
-#warning TODO: port me to OpenCV 3
-#endif
-
-#if ((CV_MAJOR_VERSION == 2) && (CV_MINOR_VERSION >=4) && (CV_SUBMINOR_VERSION>=4))
-
+#ifdef HAVE_OPENCV_FACE
+#include "FaceRecognizer.h"
 #include "Matrix.h"
 #include <nan.h>
 
+#if CV_MAJOR_VERSION >= 3
+namespace cv {
+  using std::vector;
```

Comment:
Little hacks to make the OpenCV3 versions look more like the CV2 versions, to avoid ifdefing all over the place